examples/glueviz_wcs_example.ipynb

```diff
@@ -0,0 +1,227 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename_d1 = 'NIRISS_f150w_direct_dit1_nis_cal.fits'\n",
+    "filename_d2 = 'NIRISS_f150w_direct_dit2_nis_cal.fits'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter('ignore')  # Silence warnings\n",
+    "    dither1 = load_data(filename_d1)[0]  # 0 = SCI\n",
+    "    dither2 = load_data(filename_d2)[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shows the data collection.\n",
+    "dither1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Shows the data components for SCI extension.\n",
+    "dither1.components"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we create a `glue-jupyter` app and add the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app = jglue()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.add_data(dither1)\n",
+    "app.add_data(dither2);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Having loaded both SCI extensions from different dithers and know their WCS components, we can now link them. `add_link()` links the data for overlay."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.add_link(dither1, 'Right Ascension', dither2, 'Right Ascension')\n",
+    "app.add_link(dither1, 'Declination', dither2, 'Declination')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We display both dithers in separate, but connected, viewers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer1 = app.imshow(data=dither1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer2 = app.imshow(data=dither2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Meanwhile, `keep_in_sync` links the viewers' pans.\n",
+    "\n",
+    "**TODO: Still not sure if this is what we want. The goal is to sync by WCS, not pixel location.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Tom R advised not to lock down all four corners with\n",
+    "# auto aspect ratio on or bad things will happen.\n",
+    "keep_in_sync(viewer1.state, 'x_min', viewer2.state, 'x_min')\n",
```

Comment:
@astrofrog , still not sure if this is what we want. The goal is to sync by WCS, not pixel location.